modules/nf-core/truvari/bench/tests/main.nf.test.snap

```diff
@@ -0,0 +1,234 @@
+{
+    "without bed": {
+        "content": [
+            "VcfFile [chromosomes=[], sampleCount=1, variantCount=0, phased=true, phasedAutodetect=true]",
```

Comment:
Is it normal that there is no variant ?

Reply:
Probably, this is a structural variant detection tool, so I wouldn't expect it to find anything in this dataset.

Reply:
Probably, it is being fed some dummy data.

Reply:
It looks like it all 4 vcf files were identical last time too, it is testing for how well tools call structural variants, probably there is nothing suitable in the test input.